### DISCUSSION TITLE
Welcome to Metaboverse Discussions!

### DISCUSSION THREAD

**Discussion by j-berg** [Metaboverse/Metaboverse#61] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BruceEdgar** [comment]
Hi Jordan - Will Metaboverse run on a Mac now? I tried opening the Darwin 0.11.2 version on my Mac M1 but got an error message "“Metaboverse” cannot be opened because the developer cannot be verified." Please advise! Thanks.
Bruce Edgar, Univ Utah.
